deprecate-api validator: add checks for removed APIs on 1.25 and 1.26 (#208)
operator-framework/api: README.md

 return nonEmptyResults
```

#### Passing optional key/values to the validators

> Some validators accept pass optional key/values which will be used in the checks made.

Reviewer: ```suggestion
Some validators accept optional key/values which will be used in the checks made.
```

> These values ought to be global and if the value informed is not used for 1 or more

Reviewer: ```suggestion
These values ought to be global and if the value given is not used for 1 or more
```

> Some validators accept pass optional key/values which will be used in the checks made.
> These values ought to be global and if the value informed is not used for 1 or more
> validators called then, it is ignored.

Reviewer 2: ```suggestion
Validators may accept pass optional key/values which will be used in the checks made.
These values are global and if the key/value pair provided is not used for 1 or more
validators called then, it is ignored.
```

> Following an example calling the `AlphaDeprecatedAPIsValidator` which allows us to inform

Reviewer: ```suggestion
The following example callings `AlphaDeprecatedAPIsValidator`, which allows us to inform
```

> Following an example calling the `AlphaDeprecatedAPIsValidator` which allows us to inform

Reviewer: ```suggestion
The following example callings `AlphaDeprecatedAPIsValidator`, which allows us to inform
```
